schema/project-level/project-schema.json

```diff
@@ -73,7 +73,7 @@
     },
     "additionalClassifications": {
       "title": "Additional classifications",
-      "description": "One or more additional project classifications may be provided to describe the social or economic focus of the project. This classification may take place against a locally developed codelist, or a globally established codelist.",
+      "description": "One or more additional project classifications, from the [additionalClassificationsScheme codelist](https://standard.open-contracting.org/infrastructure/{{version}}/{{lang}}/reference/codelists/#additionalclassificationsscheme), may be provided to describe the social or economic focus of the project.",
```

Comment:
This field description doesn't need to be updated. It's sufficient to reference the codelist in the description of `Classification.scheme`.